File: part 5 - operators precedence/practice/Token.cs
using System;

namespace test.part_5.practice
{
    public class Token
    {
        public static string PLUS = "PLUS";
        public static string MINUS = "MINUS";
        public static string MULTIPLY = "MULTIPLY";
        public static string DIVIDE = "DIVIDE";
        public static string INT = "INT";
        public static string LPAREN = "LPAREN";
        public static string RPAREN = "RPAREN";

        public string Type { get; set; }
        public string Value { get; set; }

        public Token(string type, string value)
        {
            this.Type = type;
            this.Value = value;
        }

        public override string ToString()
        {
            return $"Type: {Type}, Value: {Value}";
        }
    }
}

File: part 9 - simple program/lesson/Token.py
class Token(object):
    (INTEGER, PLUS, MINUS, MUL, DIV, LPAREN, RPAREN, ID, ASSIGN, BEGIN, END, SEMI, DOT, EOF) = (
        'INTEGER',
        'PLUS',
        'MINUS',
        'MUL',
        'DIV',
        '(',
        ')',
        'ID',
        'ASSIGN',
        'BEGIN',
        'END',
        'SEMI',
        'DOT',
        'EOF',
    )

    def __init__(self, type, value):
        self.type = type
        self.value = value

    def __str__(self):
        """String representation of the class instance.
        Examples:
            Token(INTEGER, 3)
            Token(PLUS, '+')
            Token(MUL, '*')
        """
        return 'Token({type}, {value})'.format(
            type=self.type,
            value=repr(self.value),
        )

    def __repr__(self):
        return self.__str__()


RESERVED_KEYWORDS = {
    'BEGIN': Token('BEGIN', 'BEGIN'),
    'END': Token('END', 'END'),
}
